pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -591,7 +591,7 @@ func PublishMetricsToZedCloud(ctx *zedagentContext, cpuMemoryStat [][]string,
 		ReportAppMetric.Memory.AvailPercentage = availableMemoryPercent
 
 		appInterfaceList := aiStatus.GetAppInterfaceList()
-		log.Debugf("ReportMetrics: domainName %s ifs %v\n",
+		log.Infof("XXX ReportMetrics: domainName %s ifs %v\n",
```

Comment:
Will we keep spewing this? If yes - is Info a good level to be?

Reply:
I'll fix those before merging. Thanks for reminding me.